metacat-main/src/test/resources/search/mapping/metacat.json

```diff
@@ -15,71 +16,47 @@
     },
     "mappings": {
         "_default_": {
-            "_timestamp" : {
-                "enabled" : true,
-                "store" : true
-            },
             "_all": { "enabled":  true },
             "properties": {
                 "audit": {
                     "properties": {
                         "createdBy": {
-                            "type": "string",
-                            "index": "not_analyzed"
+                            "type": "keyword",
+                            "index": true
                         },
                         "lastModifiedBy": {
-                            "type": "string",
-                            "index": "not_analyzed"
+                            "type": "keyword",
+                            "index": true
                         }
                     }
                 },
                 "serde": {
                     "properties": {
                         "inputFormat": {
-                            "type": "string",
-                            "fields": {
-                                "raw": {
-                                    "type": "string",
-                                    "index": "not_analyzed"
-                                }
-                            }
+                            "type": "keyword",
+                            "index": true
```

Comment:
We should set this to:
{
  "inputFormat": {
    "type" "text",
   "index" = true,
    "fields": {
      "keyword": {
        "type": "keyword",
        "ignore_above": 256
      }
    }
  }
}

Reply:
Same applies to outputformat, owner, uri and serializationLib